common/tailwind/tailwind.css

```diff
@@ -1,4 +1,9 @@
 @import 'tailwindcss';
 
-/* if this shows a warning, cahnge the type of file in vscode to 'tailwind CSS' */
+/* if this shows a warning, change the type of file in vscode to 'tailwind CSS' */
 @config "./tailwind.config.js";
+
+@theme {
+    --text-xxs: 0.625rem;
```

Comment:
Would prefer 2xs as well, but I see this is just you extracting this from tailwind.config.js, so that's all right, but you might wanna change it later